scripts/pianoroll_ghost.py

```diff
@@ -0,0 +1,96 @@
+from dolphin import controller, event, gui, memory, utils
+import mkw_core as core
+import mkw_classes as classes
+from framesequence import FrameSequence
+
+
+def set_ghost_buttons(inputs):
+    ghostcontrolleraddress = {"RMCE01": 0x809b8f70, "RMCP01": 0x809BD730,
+                              "RMCJ01": 0x809bc790, "RMCK01": 0x809abd70}
+    ghostcontrollerptr = core.chase_pointer(
+        ghostcontrolleraddress[core.get_game_id()], [0xC, 0x4, 0x48, 0x4], 'u32')
```

Comment:
In `mkw_classes.py`, add the following class definition (along with autopep8):
```py
class InputMgr:
    @staticmethod
    def chain():
        address = {"RMCE01": 0x809b8f70, "RMCP01": 0x809bd730, "RMCJ01": 0x809bc790, "RMCK01": 0x809abd70}
        return address[get_game_id()]
```

This can then be simplified into (assuming autopep8 compliance):
```suggestion
    ghost_controller = core.chase_pointer(classes.InputMgr.chain(), [0xC, 0x4, 0x48, 0x4], 'u32')
```

The variable names that reference `ghostcontrollerptr` would also need to be changed.

Reply:
This is still missing the note.